tests/commands/test_invoke_upload.py

```diff
@@ -32,9 +32,14 @@ def test_upload_raise_Z_option(mocker):
     upload_sender = mocker.patch.object(
         UploadSender, "send_upload_data", return_value=result
     )
+    upload_collector = mocker.patch.object(UploadCollector, "generate_upload_data")
```

Comment:
I think this makes sense to mock anyway.  What was previously generating the fake test coverage data?

Reply:
OK, interesting.  So it was probably searching the filesystem of `codecov-cli`'s CI or something for it's own test coverage